rest/route/service.go

```diff
@@ -115,6 +115,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/tasks/{task_id}/abort").Version(2).Post().Wrap(checkUser).RouteHandler(makeTaskAbortHandler(sc))
 	app.AddRoute("/tasks/{task_id}/generate").Version(2).Post().RouteHandler(makeGenerateTasksHandler(sc, opts.QueueGroup))
 	app.AddRoute("/tasks/{task_id}/generate").Version(2).Get().RouteHandler(makeGenerateTasksPollHandler(sc, opts.QueueGroup))
+	app.AddRoute("/tasks/{task_id}/manifest").Version(2).Get().Wrap(checkUser).RouteHandler(makeGetManifestHandler(sc)).Get()
```

Comment:
This probably doesn't require auth. There isn't any sensitive information and the UI route doesn't require auth